packages/adp-tooling/src/writer/options.ts

```diff
@@ -178,21 +182,35 @@ function addOpenSourceMiddlewares(ui5Config: UI5Config, config: AdpWriterConfig)
  * @returns list of required tasks.
  */
 function getAdpCloudCustomTasks(config: AdpWriterConfig & { target: AbapTarget } & { app: CloudApp }): CustomTask[] {
+    let target: CloudCustomTaskConfigTarget;
+    if (config.target.destination) {
+        target = {
+            destination: config.target.destination,
+            url: config.target?.url ?? vscodeUrl
+        };
+    } else {
+        target = {
+            url: config.target.url,
+            authenticationType: config.target.authenticationType,
```

Comment:
I would add the authentication to the destination config as well. As local preview will read this value when spinning up a local proxy.

Reply:
For BAS we don't need authentication type.

Reply:
Sorry, I resolved this based on an offline conversation.

Can you confirm if you think the authenticationType needs to be appended? I think if you are porting apps from BAS to VSC then the authenticationType can play a role.